PythonVisualizations/SimpleQueue.py

```diff
@@ -8,60 +8,85 @@
 
 WIDTH = 800
 HEIGHT = 400
-
 CELL_SIZE = 50
 ARRAY_X0 = 100
 ARRAY_Y0 = 100
```

Comment:
Use class attributes instead of global variables to avoid name conflicts with other modules.